src/BusinessLogic/Controllers/RegistrationController.php

```diff
@@ -79,7 +79,7 @@ public function register(array $payload)
         $brand = $this->getBrandConfigurationService()->get();
 
         $payload['platform'] = $brand->platformCode;
-        $payload['language'] = $this->getLanguage();
+        $payload['language'] = $this->getLanguage($payload['platform_country']);
```

Comment:
hi, what would be the language when the platform_country is 'UN' ?

Reply:
It will default to the English language.